components/lib/divider/Divider.spec.js

```diff
@@ -2,84 +2,18 @@ import '@testing-library/jest-dom';
 import { render } from '@testing-library/react';
 import { Divider } from './Divider';
 
-describe('Divider', () => {
-    test('when component has no properties it returns with default class', () => {
-        // Arrange
-        const { container } = render(<Divider />);
-
-        // Act + Assert
-        expect(container.getElementsByClassName('p-divider p-component p-divider-horizontal p-divider-solid p-divider-left').length).toBe(1);
-    });
-    test('when layout and align as property it returns with class', () => {
-        // Arrange
-        const { container } = render(<Divider layout={'horizontal'} align={'left'} />);
-
-        // Act + Assert
-        expect(container.getElementsByClassName('p-divider p-component p-divider-horizontal p-divider-left').length).toBe(1);
-    });
-    test('when layout and align as property it returns with class', () => {
-        // Arrange
-        const { container } = render(<Divider layout={'horizontal'} align={'right'} />);
-
-        // Act + Assert
-        expect(container.getElementsByClassName('p-divider p-component p-divider-horizontal p-divider-right').length).toBe(1);
-    });
-    test('when layout and align as property it returns with class', () => {
```

Comment:
Did you notice you repeated the same test message? It is good for the test message to reflect the case you are testing.